bridges/SimpleLoginBridge.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+class SimpleLoginBridge extends BridgeAbstract {
+	const NAME = 'SimpleLogin Blog';
+	const URI = 'https://simplelogin.io';
+	const DESCRIPTION = 'Gets the latest SimpleLogin blog posts';
+	const MAINTAINER = 'tblyler';
+	const CACHE_TIMEOUT = 3600; // 1 hour
+	const BLOG_URI = 'https://simplelogin.io/blog';
+
+	public function collectData() {
+		$html = getSimpleHTMLDOMCached(self::BLOG_URI, self::CACHE_TIMEOUT);
+		if (!$html) {
+			returnServerError('Unable to load posts from "' . self::BLOG_URI . '"!');
+		}
```

Comment:
An `or` can be used here instead of an `if`
```suggestion
		$html = getSimpleHTMLDOMCached(self::BLOG_URI, self::CACHE_TIMEOUT)
			or returnServerError('Unable to load posts from "' . self::BLOG_URI . '"!');
```